### DISCUSSION TITLE
How to use EditingWidget

### DISCUSSION THREAD

**Discussion by TheR7angelo** [Mapsui/Mapsui#2592] [Q&A]
Good morning,
In the example provided to use EditingWidget you needed the mapControl and the EditManager or in the nuget version you also need EditManipulation.

Is it possible to update the wpf document?
I don't understand what to do.

How to redo, as in the documentation, the movement of a point with a visualization of the point in movement.

**Comment by pauldendulk** [comment]
The EditingWidget in v4 should have been marked as 'experimental'. In its current state it can not be used without understanding the underlying code. The best way to go forward is to copy the code of the editing widget and related components to your own code and rewrite it to your own needs.

The EditingWidget is cross platform so we do not have any specific documentation WPF. Also instead of investing in better documentation about the current EditingWidget I prefer to rewrite the it into an EditingWidget that is easier to understand. In v5 things have improved but it needs further improvement and I still consider to move most of the related code back to the samples.
